src/providers/bitpay-card/bitpay-card.ts

```diff
@@ -102,6 +108,10 @@ export class BitPayCardProvider {
     });
   }
 
+  logEvent(eventName: string, eventParams: { [key: string]: any }) {
```

Comment:
We currently have the `analyticsProvider` ... why don't we use that directly to make the `logEvents`?